app/controllers/protocols_controller.rb

```diff
@@ -98,6 +97,19 @@ def update_protocol_type
     respond_to :js
   end
 
+  def validate_rmid
+    respond_to :js
+
+    if protocol_params[:research_master_id]
+      @protocol = Protocol.new(protocol_params)
+      @protocol.valid?
+
+      unless (@errors = @protocol.errors.messages[:base]).any?
+        @rmid_record = Protocol.get_rmid(protocol_params[:research_master_id])
+      end
+    end
+  end
+
```

Comment:
This is the new route that checks the RMID by calling .`valid?` to run RMID model validations